docs/api/wrappers/pz_wrappers.md

```diff
@@ -90,6 +91,7 @@ for agent in env.agent_iter():
         action = None
     else:
         action = env.action_space(agent).sample()  # this is where you would insert your policy
+    #TODO RPS observe() returns an np.array instead of dict, triggering an assert statement in TerminateIllegalWrapper
```

Comment:
Jeez both of the examples converting envs fail if you run them that’s embarrassing, never took the time to test myself but whoever wrote them clearly did a bad job. If you could try to get to the bottom of the issue here that would be great too

Reply:
Sure I'll take a look

Reply:
Oh ok so rps is no action mask since it’s so simple, terminate illegal should be able to handle things like that though and give a warning like “Environment has no illegal action mask, terminating only for invalid actions” and check both obs[action mask] and info[agent][action mask]

The aec docs have an example with both, if you could check that info is indexed by agent first that would be good. Shimmy’s OpenSpiel wrapper may not use the correct info format in which case I’ll update it. 

Also there’s probably other code which hard codes the action mask to be obs, we want it to be flexible so if you could take a look at other uses and make sure they account for info as an option too.

Reply:
Looks like openspiel's action mask is just in info[action_mask] as I thought, adapted it in my new pr